src/main/java/com/todoist/markup/MarkupFlags.java

```diff
@@ -0,0 +1,13 @@
+package com.todoist.markup;
+
+public class MarkupFlags {
```

Comment:
I'm not a big fan of this move. I understand the reasoning behind having all constants together in a separate class, but in this case only `MarkupParser` understands and handles them. For this reason, I still think they should all be there.